open_academy/i18n/es_MX.po

```diff
@@ -0,0 +1,342 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* open_academy
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-09-07 20:45+0000\n"
+"PO-Revision-Date: 2022-09-07 20:45+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.session_template
+msgid "<u>Session Info Report</u>"
+msgstr "<u>Reporte de informes de sesión</u>"
```

Comment:
Keep the same style between the terms if you use Upper case use in the translated term 
```suggestion
msgstr "<u>Reporte de Informes de Sesión</u>" 
```

or 

````
msgstr "<u>Reporte de Info de Sesión</u>" 
````
Also why Session Info Report? Is it something redundant not? Just Session Report for me its cleaner